pkg/deploy/gateway/oauth_proxy.go

```diff
@@ -99,6 +99,30 @@ skip_provider_button = false
 		skipAuthConfig(ctx.CheCluster))
 }
 
+func getSecretValue(ctx *chetypes.DeployContext) string {
```

Comment:
1. Please add test case into `oauth_proxy_test.go`
Use the following code to initialize deploy context:
```go
	ctx := test.GetDeployContext(
		&chev2.CheCluster{
			Spec: chev2.CheClusterSpec{
				Networking: chev2.CheClusterSpecNetworking{
					Auth: chev2.Auth{
						OAuthSecret: "my-secret",
					},
				}},
		},
		[]runtime.Object{
			&corev1.Secret{
				TypeMeta: metav1.TypeMeta{
					Kind:       "Secret",
					APIVersion: "v1",
				},
				ObjectMeta: metav1.ObjectMeta{
					Name:      "my-secret",
					Namespace: "eclipse-che",
					Labels:    map[string]string{constants.KubernetesPartOfLabelKey: constants.CheEclipseOrg},
				},
				Type: corev1.SecretTypeOpaque,
				Data: map[string][]byte{"oAuthSecret": []byte("my")},
			},
		})
```

2. Update the description of `OAuthSecret` field in `v2/checluster_types.go`
Note, the the secret must have a `app.kubernetes.io/part-of=che.eclipse.org` label. 
Please, mention this in the description as well.

Reply:
Thank you! 4 test cases created, logging removed, and user documentation updated in `v2/checluster_types.go`.